WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -219,8 +219,10 @@ private void initViewModel() {
             LoginNavigationEvents loginEvent = event.getContentIfNotHandled();
             if (loginEvent instanceof ShowSiteAddressError) {
                 showSiteAddressError((ShowSiteAddressError) loginEvent);
+            } else if (loginEvent instanceof ShowNoJetpackSites) {
+                showNoJetpackSites();
             } else {
-                showNoJetpackSitesError((ShowNoJetpackSitesError) loginEvent);
+                // NO OP
```

Comment:
Minor (🔍): This `else` block is effectively not needed any more. Consider removing it altogether.